src/main/kotlin/com/ampnet/reportservice/service/impl/TemplateDataServiceImpl.kt

```diff
@@ -158,4 +169,17 @@ class TemplateDataServiceImpl(
                 ErrorCode.INT_REQUEST, "Transaction doesn't belong to user wallet with hash: $txHash"
             )
     }
+
+    private fun getTranslations(userLanguage: String): Translations {
+        val json = javaClass.classLoader.getResource("templates/translations.json")?.readText()
+        val typeRef = object : TypeReference<Map<String, Map<String, String>>>() {}
+        val allTranslations = objectMapper.readValue<Map<String, Map<String, String>>>(json, typeRef)
+        val translations: HashMap<String, String> = HashMap()
+        allTranslations.forEach { (key, map) ->
+            map.forEach { (language, translation) ->
+                if (language.equals(userLanguage, true)) translations[key] = translation
+            }
+        }
+        return Translations.from(translations)
+    }
```

Comment:
This is a very bad solution because you load json file and map it every time you need translations. With the current implementation, this is for every requested user report, this process is started and loads translations.

Service should lazy load a translations json file when it's first needed and stores a list of all available translations/languages. After first and only loading of translations, it just serves translations for the requested language.

Map values without typeRef:
```kotlin
        val allTranslations: Map<String, Map<String, String>> = objectMapper.readValue(json)
```